TrafficCrash/Pages/Index.cshtml.cs

```diff
@@ -24,28 +24,23 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public JsonResult OnGet()
         {
-            List<Vehicle_People_Crash_details> vehicle_n_people = new List<Vehicle_People_Crash_details>();
+            List<VehiclePeopleCrashdetails> vehicle_n_people = new List<VehiclePeopleCrashdetails>();
```

Comment:
This is more in line with naming conventions. While you're in here, change the vehicle_n_people to camelCase as well.